virttest/utils_conn.py

```diff
@@ -1106,17 +1129,30 @@ def server_setup(self, on_local=False):
                 raise ConnSCPError('AdminHost', local_path,
                                    server_ip, remote_path, detail)
         # When qemu supports TLS, it needs not to modify below
-        # configuration files, so simply return.
+        # configuration files, so simply return
         if self.qemu_tls:
             return
 
         # Ensure to use proper configuration objects
         if on_local:
             operate_libvirtdconf = self.client_libvirtdconf
             operate_syslibvirtd = self.client_syslibvirtd
+            operate_qemuconf = self.server_qemuconf
```

Comment:
why do we use server qemu conf for local host configuration ?

Reply:
on_local means the client is also on the server machine and it's the same one.